aroquev/src/main/webapp/style.css

```diff
@@ -1,3 +1,36 @@
+body {
+  font-family: 'Noto Sans JP', sans-serif;
+}
+
+/* For the header */
+/* Add a black background color to the top navigation */
+.topnav {
+  background-color: #333;
+  overflow: hidden;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+  float: left;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+/* Change the color of links on hover */
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+/* Add a color to the active/current link */
+.topnav a.active {
```

Comment:
You'll need to have some JS that applies this class to the nav item that is clicked. Right now the "active" item is always "Home"

Reply:
Right, I left it like that because I thought red looked cool on the Home button hehe, but once I divide it in more pages, it will change to the page you're in.